docs/pguidance/agent-install-packages.md

```diff
@@ -0,0 +1,142 @@
+# Install using *nix packages
```

Comment:
Is the asterisk next to nix indication of a footnote? Not sure if this would work, but try formatting this as a footnote: https://www.markdownguide.org/extended-syntax/#footnotes

Reply:
This is the Splunk preferred style for Unix.